test/bigtest/tests/viewInterface.e2e.js

```diff
@@ -0,0 +1,85 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+
+import { InterfacesPaneInteractor } from '../interactors';
+
+describe('View interface', () => {
+  setupApplication();
+
+  const viewInterface = new InterfacesPaneInteractor();
+  let orgInterface;
+
+  beforeEach(function () {
+    orgInterface = this.server.create('interface');
+
+    return this.visit(`/organizations/interface/${orgInterface.id}/view`, () => {
+      expect(viewInterface.$root).to.exist;
+    });
+  });
+
+  describe('actions', () => {
+    it('should be present', () => {
+      expect(viewInterface.actions.isPresent).to.be.true;
+    });
+
+    describe('click unassign button', () => {
```

Comment:
does it make sense to make such nesting?

Reply:
for me there are two section - actions and view and I decided to separate them for different blocks, but now I rearranged test cases without nesting.